src/runtime/link-accounts-flow.js

```diff
@@ -18,10 +18,10 @@ import {ActivityIframeView} from '../ui/activity-iframe-view';
 import {getHostUrl} from '../utils/url';
 
 const LINK_FRONT_IFRAME_URL =
-    '$frontend$/subscribewithgoogleclientui/linkfrontiframe$frontendDebug$';
+    '$frontend$/swglib/linkfrontiframe$frontendDebug$';
```

Comment:
Why change this url to swglib?

Reply:
It's been there for a while - just a lot shorter...